data/_ui/tooltips.txt

```diff
@@ -406,22 +418,22 @@ tip "outfit scan opacity:"
 	`Increases the time required for other ships to perform an outfit scan of this ship. A value of one has the same influence on the time taken as adding one more ton of outfit space to scan.`
 
 tip "outfit space added:"
-	`How much general-purpose space this outfit provides.`
+	`Tons of general-purpose space this outfit provides.`
 
 tip "outfit space needed:"
-	`How much general-purpose space this outfit uses up. Some outfits also use up weapon space or engine space.`
+	`Tons of general-purpose space this outfit uses up. Some outfits also use up weapon space or engine space.`
 
 tip "outfit space free:"
-	`How much space this ship has for outfits (including weapons and engines).`
+	`Tons of space this ship has for outfits (including weapons and engines).`
 
 tip "overheat damage rate:"
-	`How much hull damage this ship will take per second that it is overheated. Overheat damage scales with how far past the overheat threshold the ship is.`
+	`Hull damage taken per second when overheated. Overheat damage scales with how far past the overheat threshold the ship is.`
 
 tip "overheat damage threshold:"
 	`Increases the percentage past overheated at which point overheat damage begins.`
 
 tip "piercing protection:"
-	`Protection against weapons that pierce shields. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in damage leak, while a total value of 11 only results in a 10 percent reduction.`
+	`Protection provided against shield piercing damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in damage leak, while a total value of 11 only results in a 10 percent reduction.`
```

Comment:
Piercing isn't an actual damage type, it's a modifier to shield and hull damage, so I don't think that calling it "shield piercing damage" makes sense. It's also not protection in the same way that the other protection attributes reduce the total amount of damage received, as piercing protection just bounces damage that would've hit the hull back to the shields. We could probably also use a different term than "damage leak," or maybe explain what is meant by that (although that may be redundant with reading piercing's tooltip).

Perhaps look to piercing resistance's tooltip.

Reply:
Piercing resistance/protection doesn't work with shield disruption, right? So if I say something like "protection provided against damage bypassing shields", that would be misleading.

~~This is hard.~~

```suggestion
	`Protection provided against shield piercing damage. Unlike other protection attributes, this attribute bounces the incoming hull damage to your shields, but doesn't reduce the overall damage dealt. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in damage leak, while a total value of 11 only results in a 10 percent reduction.`
```

I honestly can't think of a better name for piercing "damage".

Reply:
Feels too wordy. I think the problem is the use of the word damage, because like I said, this isn't actually a damage type.
```suggestion
	`Protection provided against piercing. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in piercing, while a total value of 11 only results in a 10 percent reduction.`
```